pydantify/models/base.py

```diff
@@ -95,6 +95,21 @@ def __init__(self, stm: Statement):
         self.comments: str | None = __class__.__extract_comments(stm)
         self.description: str | None = __class__.__extract_description(stm)
 
+        if self.keyword == "list":
```

Comment:
Could we also do this in `class ListNode(Node):`?
https://github.com/pydantify/pydantify/blob/0a9693046718581aeb48add434c68d51e3cbe7b7/pydantify/models/models.py#L174